staff/elena-rodriguez/precourse/airlinesPro/index.js

```diff
@@ -0,0 +1,218 @@
+var flights = [
+    { id: 00, to: 'Bilbao', from: 'Barcelona', cost: 1600, scale: false },
+    { id: 01, to: 'New York', from: 'Barcelona', cost: 700, scale: false },
+    { id: 02, to: 'Los Angeles', from: 'Madrid', cost: 1100, scale: true },
+    { id: 03, to: 'Paris', from: 'Barcelona', cost: 210, scale: false },
+    { id: 04, to: 'Roma', from: 'Barcelona', cost: 150, scale: false },
+    { id: 05, to: 'London', from: 'Madrid', cost: 200, scale: false },
+    { id: 06, to: 'Madrid', from: 'Barcelona', cost: 90, scale: false },
+    { id: 07, to: 'Tokyo', from: 'Madrid', cost: 1500, scale: true },
+    { id: 08, to: 'Shangai', from: 'Barcelona', cost: 800, scale: true },
+    { id: 09, to: 'Sydney', from: 'Barcelona', cost: 150, scale: true },
+    { id: 10, to: 'Tel-Aviv', from: 'Madrid', cost: 150, scale: false }
+];
+
+
+function printNice (flights) { 
+    console.log('Estos son los vuelos para hoy: ')
+    for (let op = 0; op < flights.length; op++) {
+        if(flights[op].scale === false) {
+        console.log('Id: ' + flights[op].id + '.El vuelo con origen: ' + flights[op].from + ' y con destino: ' + flights[op].to + 
+        ' tiene un coste de ' + flights[op].cost + '€, y no realiza ninguna escala.')}
+        else {console.log('Id: ' + flights[op].id + '.El vuelo con origen: ' + flights[op].from + ' y con destino: ' + flights[op].to + 
+        ' tiene un coste de ' + flights[op].cost + '€, y realiza una escala.')}
+    }
+}
+
+function averageCost(flights) {
+    var resultAdd = 0; 
+    for (let op = 0; op < flights.length; op++) {
+        resultAdd += flights[op].cost;
+    }
+    resultAdd = resultAdd / flights.length;
+    resultAdd = resultAdd.toFixed(2);
+    console.log('El coste medio de los vuelos es de: ' + resultAdd + '€');
+}
+
+function vuelosEscalas(flights){
+
+console.log('Vuelos con escala:');
+for (let op = 0; op < flights.length; op ++) {
+    if (flights[op].scale === true) {
+        console.log('Origen: ' + flights[op].from + '; Destino: ' + flights[op].to + '; Precio: ' + flights[op].cost + '€.')
+    }
+}
+}
+
+function lastFlights(flights) {
+console.log('Últimos vuelos del día: ')
+for (let op = 6; op < flights.length; op++) {
+    console.log('Vuelo con origen: ' + flights[op].from + ' y con destino: ' + flights[op].to)}
+}
```

Comment:
This function could only work correctly if you have 10 flights in the array, maybe you could use the **slice** method
example:
`var lastFlights = flights.slice(-5)`

Reply:
Perfect, fixed!!